nservicebus/azure-service-bus/multiple-namespaces-support.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Multiple namespace support
+summary: Configuring Azure ServiceBus transport to support different partitioning strategies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+reviewed: 2016-04-18
+---
+
+`Azure Service Bus` transport enables different _namespace partitioning strategies_ to cover different scenarios such as High Availability and multiple Data Center support or to overcome [service limits](https://azure.microsoft.com/en-us/documentation/articles/service-bus-quotas/).  
+
+Configuration APIs give the possibility to select desired _namespace partitioning strategies_ and then to configure specific namespace(s) to use.
+
+## Single namespace partitioning ##
+
+The default _namespace partitioning strategy_ is `SingleNamespacePartitioning`. With the default setup, transport uses only a single namespace to send and receive messages.  
+Namespace connection string can be defined via code using `ConnectionString` extension:
+
+snippet: single_namespace_partitioning_strategy_with_default_connection_string
+
+or selecting explicitly the `SingleNamespacePartitioning` strategy and providing a map between namespace name and namespace connection string.  
+Transport uses namespace name to avoid sharing namespace secrets.
+
+snippet: single_namespace_partitioning_strategy_with_add_namespace
+
+With this strategy, only one namespace has to be configured. Configuration throws a `ConfigurationErrorsException` if more than one namespace has been configured.   
+`SingleNamespacePartitioning` strategy is the easiest one but it's the less relyable: the only configured namespace is a single point of failure for the entire system.
```

Comment:
> what do you mean by that? easiest to configure? is that a default option?

`SingleNamespacePartitioning` is the default option, and it's the easiest strategy implementation because it cover simple scenario: read/write from a single namespace. No HA or possibility to scale for instance